c/tskit/tables.h

```diff
@@ -712,6 +712,13 @@ typedef struct {
 /* Flags for table collection init */
 #define TSK_NO_EDGE_METADATA (1 << 0)
 
+/* Flags for table collection load */
+/* This shares an interface with table collection init, so we have to
+ * increment offset by one.
+ * NOTE: Should these two be under the same header?
```

Comment:
Good questions - let's do this as part of #1720

Can you leave a comment here please, "TODO review as part of https://github.com/tskit-dev/tskit/issues/1720"